CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 ## 6.20.0 (Unreleased)
 
+NOTES:
+* compute: `google_compute_firewall_policy` now uses MMv1 engine instead of DCL. ([#9228](https://github.com/hashicorp/terraform-provider-google-beta/pull/9228))
+
+FEATURES:
+* **New Data Source:** `google_beyondcorp_application_iam_policy` ([#9205](https://github.com/hashicorp/terraform-provider-google-beta/pull/9205))
+* **New Data Source:** `google_parameter_manager_parameter_version_render` ([#9190](https://github.com/hashicorp/terraform-provider-google-beta/pull/9190))
+* **New Data Source:** `google_parameter_manager_regional_parameter_version_render` (beta) ([#9232](https://github.com/hashicorp/terraform-provider-google-beta/pull/9232))
+* **New Resource:** `google_beyondcorp_application` ([#9205](https://github.com/hashicorp/terraform-provider-google-beta/pull/9205))
+* **New Resource:** `google_beyondcorp_application_iam_binding`  ([#9205](https://github.com/hashicorp/terraform-provider-google-beta/pull/9205))
+* **New Resource:** `google_beyondcorp_application_iam_member`  ([#9205](https://github.com/hashicorp/terraform-provider-google-beta/pull/9205))
+* **New Resource:** `google_beyondcorp_application_iam_policy`  ([#9205](https://github.com/hashicorp/terraform-provider-google-beta/pull/9205))
+* **New Resource:** `google_bigquery_analytics_hub_listing_subscription` ([#9195](https://github.com/hashicorp/terraform-provider-google-beta/pull/9195))
+* **New Resource:** `google_colab_notebook_execution` ([#9186](https://github.com/hashicorp/terraform-provider-google-beta/pull/9186))
+* **New Resource:** `google_colab_schedule` ([#9226](https://github.com/hashicorp/terraform-provider-google-beta/pull/9226))
+* **New Resource:** `google_compute_network_firewall_policy_packet_mirroring_rule` ([#9202](https://github.com/hashicorp/terraform-provider-google-beta/pull/9202))
+* **New Resource:** `google_gemini_logging_setting` ([#9198](https://github.com/hashicorp/terraform-provider-google-beta/pull/9198))
+* **New Resource:** `google_gemini_release_channel_setting` ([#9207](https://github.com/hashicorp/terraform-provider-google-beta/pull/9207))
+
+IMPROVEMENTS:
+* accesscontextmanager: added `resource` to `sources` in `egress_from` under resources `google_access_context_manager_service_perimeter`, `google_access_context_manager_service_perimeters`, `google_access_context_manager_service_perimeter_egress_policy`, `google_access_context_manager_service_perimeter_dry_run_egress_policy` ([#9196](https://github.com/hashicorp/terraform-provider-google-beta/pull/9196))
+* appengine: added a mitigation for an upcoming default change to `standard_scheduler_settings.max_instances` for new `google_app_engine_standard_app_version` resources. If the field is not specified in configuration, diffs will now be ignored. ([#9233](https://github.com/hashicorp/terraform-provider-google-beta/pull/9233))
```

Comment:
Should this be under bug instead of improvement? Not really sure

Reply:
probably - I'll move it.